modules/windows-linux-containers-differences.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-containers.adoc
+
+[id="windows-linux-containers-differences_{context}"]
+= Windows container and Linux container differences
+
+_WIP_
```

Comment:
Is this intended to be merged as-is?

Reply:
No, I will remove it if I don't get info on it. This one won't be required for the initial release, but would certainly be useful to have.